GermanFilter/sections/specific.txt

```diff
@@ -2464,6 +2464,7 @@ wiwo.de##.hcf-partner
 merkurist.de##shipment
 merkurist.de##shipment-premium
 merkurist.de##div[sponsored-article-impression]
+merkurist.de#$?#div[sparticle-impression-article], div[sparticle-impression] { visibility: hidden !important; }
```

Comment:
As I see, extended CSS is not required here. Should work with `#$#`.

Also we avoid using comma separated selectors, because if the rule will break something, it takes more time to find a reason. Please divide into two rules.

Reply:
@Alex-302 This looks like native ads, should we block this at all?

Reply:
Yes, we can. But on some sites they have interesting content.